src/AI/WallyAI.java

```diff
@@ -786,22 +889,22 @@ private double findAssaultKills(GameMap gameMap, Queue<Unit> unitQueue, Map<XYCo
           neededAttacks.put(xyc, unit);
           double thisDamage = CombatEngine.simulateBattleResults(unit, target, gameMap, xyc.xCoord, xyc.yCoord).defenderHPLoss;
 
-          if (thisDamage > target.getPreciseHP())
+          if( thisDamage > target.getPreciseHP() )
             continue; // OHKOs should be decided using different logic
 
-          log(String.format("  Use %s to deal %sHP?", unit.toStringWithLocation(), thisDamage));
+//          log(String.format("  Use %s to deal %sHP?", unit.toStringWithLocation(), thisDamage));
```

Comment:
Are these log statements supposed to be commented out, or just removed?

Reply:
I really thought about deleting them, but it'd be a real pain to get them back if I ever wanted to debug that stuff more.
The system's pretty well working as I expect though, so they're mostly log clutter right now.
If we had log levels, they'd be like `AI_TRACE` or something.